### ISSUE TITLE
Pickaxe Animation feels Weightless (Possible Bug)

### ISSUE THREAD

**Issue by ifozz** [anegostudios/VintageStory-Issues#4286]
### Game Version

v1.20.0 pre-5

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

The pickaxe animation now lacks any weight and looks like you are just gingerly tapping the rock until it breaks.
I thought I would write a report as it does not look right and so I suspected this may be unintended behaviour.

### How to reproduce

1. Mine something with a pickaxe.

### Logs

```Log```

**Comment by Craluminum2413** [comment]
Wasn't it always was like this?

**Comment by ifozz** [comment]
> Hasn't it always been like this?

Nope, it used to be an overhead swing. If you test it out in 1.20, it's now this strange gentle tapping animation.
The old 1.19 one had more weight to it.
